### PR TITLE
[zh-cn]Update blogs-case-studies.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/contribute/new-content/blogs-case-studies.md [hunk 1]
```diff
@@ -147,15 +147,15 @@ To submit a blog post, follow these steps:
...
   - Posts about other CNCF projects may or may not be on topic. We recommend asking the blog team before submitting a draft.
     - Many CNCF projects have their own blog. These are often a better choice for posts. There are times of major feature or milestone for a CNCF project that users would be interested in reading on the Kubernetes blog.
   - Blog posts about contributing to the Kubernetes project should be in the [Kubernetes Contributors site](https://kubernetes.dev)
 -->
 - 博客内容应该对 Kubernetes 用户有用。
   - 与参与 Kubernetes SIGs 活动相关，或者与这类活动的结果相关的主题通常是切题的。
-    请参考[上游推广团队](https://github.com/kubernetes/community/blob/master/communication/marketing-team/blog-guidelines.md#upstream-marketing-blog-guidelines)的工作以获得对此类博文的支持。
+    请参考[上游推广团队](https://github.com/kubernetes/community/blob/master/communication/marketing-team/storytelling-resources/blog-guidelines.md#upstream-marketing-blog-guidelines)的工作以获得对此类博文的支持。
   - Kubernetes 的组件都有意设计得模块化，因此使用类似 CNI、CSI 等集成点的工具
     通常都是切题的。
```
Comment (windsonsea) [inline content/zh-cn/docs/contribute/new-content/blogs-case-studies.md:160]:
```suggestion
  - Kubernetes 的组件都有意设计得模块化，因此使用类似 CNI、CSI 等集成点的工具通常都是切题的。
```

**Review by windsonsea — COMMENTED** [review]
Would you mind to check through this topic rather than too many size-xs flooding your PRs list, which is easy to check by everyone.

**Comment by Sea-n** [comment]
Hey @kinzhi,

Would you like to address the previous concern?
Please let us know what you think about it using the comment function.

**Comment by Sea-n** [comment]
Thanks for purposing this PR. But since you didn't respond for a while, we have to close this PR now.
/close

Feel free to `/reopen` when you are ready to move forward.
(Note that please don't push anything while PR closed, or you won't be able to reopen it.)

**Pull request closed without merging**